### ISSUE TITLE
我基于该项目实现了pytorch的版本

### ISSUE THREAD

**Issue by wireless911** [chiayewken/Span-ASTE#12]
hello！
      感谢您开源出来的代码和数据，让我们可以进行研究和试验，鉴于allennlp我不是很熟悉，我在基于这个存储库实现了lstm版本的span-aste，我想可能会有人跟我会碰到同样的问题，我将我的代码也开源了出来，希望可以帮到更多的人，期待您提出宝贵的建议 。https://github.com/wireless911/span-aste.git

                                                                                                                                                                                               谢谢

**Comment by chiayewken** [comment]
That's great, are you able to reproduce the result?

**Comment by wireless911** [comment]
Thank you for your reply. After the bert version I implemented was trained in the 15res dataset, the f1 value of the validation set was 69.1 and the result in the paper was still slightly different. I am still exploring.

**Comment by yedongyu1996** [comment]
太感谢了
已经让复杂的包搞得晕头转向了，总是包冲突

**Comment by AnitaSherry** [comment]
太好了，我也是各种包搞得头疼，Span-ASTE用的包都太过时了
